airflow/config_templates/default_airflow.cfg

```
@@ -1197,9 +1197,14 @@ delete_worker_pods_on_failure = False
 worker_pods_creation_batch_size = 1
 
 # Allows users to launch pods in multiple namespaces.
-# Will require creating a cluster-role for the scheduler
+# Will require creating a cluster-role for the scheduler, or use namespace_list configuration.
 multi_namespace_mode = False
 
+# If multi_namespace_mode is True while scheduler does not have a cluster-role,
+# give the list of namespaces where the scheduler will schedule jobs
+# Scheduler needs to have the necessary permissions in these namespaces.
+namespace_list =
```

maybe this would be better.  

```suggestion
multi_namespace_mode_namespace_list =
```
it's really a sub-option of multi_namespace_mode.

wdyt?

---

Sure, that sounds sensible to me.

Let me add a follow-up commit.

---

Address with commit https://github.com/apache/airflow/pull/28047/commits/1c1f25d92e0faeb4f84069056dbdc24c1e53fa93